internal/drivers/manager.go

```diff
@@ -2,17 +2,19 @@ package drivers
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"runtime/debug"
 	"sort"
 	"sync"
 	"time"
 
+	"github.com/cenkalti/backoff/v4"
 	"github.com/drone-runners/drone-runner-aws/internal/certs"
 	"github.com/drone-runners/drone-runner-aws/store"
 	"github.com/drone-runners/drone-runner-aws/types"
 	"github.com/drone/runner-go/logger"
+	lehttp "github.com/harness/lite-engine/cli/client"
```

Comment:
import ordering

Reply:
I guess ordering is already correct. No changes are shown on go fmt